glossary.md

```diff
@@ -220,6 +227,13 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | population                         | tổng thể                                                       |                       |
 | p-value                            | trị số p                                                       |                       |
 
+## Q
+| English               | Tiếng Việt (Nếu bạn bổ sung thì KHÔNG căn chỉnh các dòng khác) | Thảo luận tại |
+|-----------------------|----------------------------------------------------------------|---------------|
+| quadratic             | toàn phương                                                    |               |
+| quadratic form        | dạng toàn phương                                               |               |
```

Comment:
Những từ thế này có quá chi tiết không nhỉ @tiepvupsu? Như từ `quy hoạch toàn phương` thì em hiểu cần đưa vô glossary vì  `programming` nghĩa thường là `lập trình` nhưng còn form em thấy không có gì đặc biệt lắm?

Reply:
Anh nghĩ không sao. Anh thêm cả cụm `quadratic` này vào vì có  bạn dịch `quadratic form` là `dạng bậc hai`.